rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton-v3.c

```diff
@@ -0,0 +1,93 @@
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/wait.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <linux/types.h>
+#include <sys/stat.h>
+#include "liberpal-skeleton.h"
+#include "sgx_call.h"
+#include "defines.h"
+
+int pal_get_version(void)
+{
+	return 3;
+}
+
+int pal_init(pal_attr_t *attr)
+{
+	return __pal_init(attr);
+}
+
+int pal_create_process(pal_create_process_args *args)
```

Comment:
Move this function into liberal-skeleton.c? I think this function is same with v2.